### ISSUE TITLE
[Nuxeo] Nuxeo feed with duplicate entries (Ramicova)

### ISSUE THREAD

**Issue by elopatin-uc3** [CDLUC3/mrt-doc#113]
Nuxeo Ramicova collection has duplicate entries which will cause problems during Ingest.

https://s3.amazonaws.com/static.ucldc.cdlib.org/merritt/ucldc_collection_26098.atom
 - [ ] Barbara Hui will fix feed
 - [ ] Test in Merritt Stage
 - [ ] Process in Merritt production

**Comment by elopatin-uc3** [comment]
Looks like Ramicova feed still has duplicates.  I'll check with Publishing group

Quick duplicate check...
   **curl --silent https://s3.amazonaws.com/static.ucldc.cdlib.org/merritt/ucldc_collection_26098.atom | grep 'dc:identifier' | sort | tail -2 ** (Mark Reyes - Apr 3, 2019)

**Comment by terrywbrady** [comment]
@elopatin-uc3 , still an issue?

**Comment by elopatin-uc3** [comment]
Yes, confirmed with Mark that the collection and feed need analysis. Leaving open
